flow/__init__.py

```diff
@@ -16,31 +16,33 @@
 from .template import init
 
 # Import packaged environments unless disabled in config:
-from .util.config import get_config_value
+from .util.config import get_config_value as _get_config_value
```

Comment:
This isn't intended to be in the public namespace -- it's only used within this file as a utility function and not listed in `__all__`.